tiled_layers/web-tiled-layer/src/main/java/com/esri/samples/web_tiled_layer/WebTiledLayerSample.java

```diff
@@ -68,10 +68,9 @@ public void start(Stage stage) {
           // set the web tiled layer as the map's basemap
           map.setBasemap(new Basemap(webTiledLayer));
           // set custom attribution on the layer
-          webTiledLayer.setAttribution("Map tiles by <a href=\"http://stamen.com/\">Stamen Design</a>, " +
-              "under <a href=\"http://creativecommons.org/licenses/by/3.0\">CC BY 3.0</a>. " +
-              "Data by <a href=\"http://openstreetmap.org/\">OpenStreetMap</a>, " +
-              "under <a href=\"http://creativecommons.org/licenses/by-sa/3.0\">CC BY SA</a>.");
+          webTiledLayer.setAttribution("Map tiles by ArcGIS Living Atlas of the World" +
+              ", under the Esri Master License Agreement." +
```

Comment:
You might want to add a space between the `.` and `Data`.

Reply:
Yes fair point, I'll add that